tmail-backend/tmail-third-party/ai-bot/src/main/java/com/linagora/tmail/mailet/AIRedactioanlHelper.java

```diff
@@ -0,0 +1,73 @@
+package com.linagora.tmail.mailet;
+
+
+import java.io.IOException;
+
+import com.google.common.base.Strings;
+import com.linagora.tmail.conf.AIBotConfigLoader;
+import dev.ai4j.openai4j.OpenAiHttpException;
+import org.apache.mailet.MailetException;
+import dev.langchain4j.data.message.ChatMessage;
+import dev.langchain4j.data.message.SystemMessage;
+import dev.langchain4j.data.message.UserMessage;
+import dev.langchain4j.model.chat.ChatLanguageModel;
+import reactor.core.publisher.Mono;
+import com.linagora.tmail.conf.AIBotProperties;
+
+public class AIRedactioanlHelper{
+    private AIBotConfig config;
+    private ChatLanguageModel chatLanguageModel;
+    private final ChatLanguageModelFactory chatLanguageModelFactory;
+
+
+
+    public AIRedactioanlHelper(ChatLanguageModelFactory chatLanguageModelFactory) {
+        this.chatLanguageModelFactory = chatLanguageModelFactory;
+    }
+    public Mono<String> suggestContent(String userInput, String mailContent) throws OpenAiHttpException, MailetException, IOException {
+        if (Strings.isNullOrEmpty(userInput) || Strings.isNullOrEmpty(mailContent)) {
+            return Mono.error(new IllegalArgumentException("User input and mail content cannot be null or empty"));
+        }
+        //configuration de bot
+        confuguerAiBot();
```

Comment:
Configuration shall be done from within the constructor and take the config as a parameter.